src/lib/flight_tasks/tasks/ManualAltitude/FlightTaskManualAltitude.cpp

```diff
@@ -61,6 +61,16 @@ bool FlightTaskManualAltitude::activate(vehicle_local_position_setpoint_s last_s
 
 	_constraints.tilt = math::radians(_param_mpc_man_tilt_max.get());
 
+	_updateConstraintsFromEstimator();
+
+	_max_speed_up = _constraints.speed_up;
+	_min_speed_down = _constraints.speed_down;
```

Comment:
```suggestion
	_max_speed_down = _constraints.speed_down;
```
The name confuses me since if I read min_speed_down I'm assuming you have to fly a certain minimum speed to descend...

Reply:
Totally agree, I made a 2nd commit to rename that.

Reply:
@MaEtUgR Can you approve again please?